enterprise/server/cmd/executor/executor.go

```diff
@@ -294,8 +296,18 @@ func main() {
 		repb.RegisterActionCacheServer(localServer, actionCacheServer)
 	}
 
+	// Setup SSL for monitoring endpoints (optional).
+	if err := ssl.Register(env); err != nil {
```

Comment:
will this fail if no certs are configured? (It seems like enabling ssl should be fully optional)

I had a look in ssl.go and it seemed possible, but I haven't run it so not 100%.

Reply:
The SSL service becomes effectively a no-op if `ssl.ssl_enabled` is false, but I moved this inside the conditional below just to make it extra clear that SSL is for monitoring only.

Reply:
oh i missed that flag. thanks for checking!